src/core/dfly_core_test.cc

```diff
@@ -65,4 +71,42 @@ TEST_F(IntentLockTest, Basic) {
   ASSERT_TRUE(lk_.Check(IntentLock::EXCLUSIVE));
 }
 
+class StringMatchTest : public ::testing::Test {
+ protected:
+  // wrapper around stringmatchlen with stringview arguments
+  int MatchLen(string_view pattern, string_view str, bool nocase) {
```

Comment:
We have a similar function called Matches :smile: Also not sure why you added tests for `stringmatchlen` since it's heavily tested in Valkey + it's used in ACL by both Valkey and us.

Reply:
I did not find any tests in valkey besides one very simple "fuzzy" test.

I covered it by tests so we could replace the implementation without breaking the behaviour.

Reply:
sounds good!